apps/mymedicare_cb/models.py

```diff
@@ -51,170 +47,246 @@ def get_and_update_user(subject, mbi_hash, hicn_hash, first_name, last_name, ema
+        log_dict["status"] = "FAIL"
+        log_dict["mesg"] = err_msg
+        logger.info(log_dict)
         raise BBMyMedicareCallbackCrosswalkCreateException(err_msg)
 
     if fhir_id == "":
         err_msg = "fhir_id can not be an empty string"
-        log_create_beneficiary_record(auth_flow_dict, "FAIL", username, fhir_id, user_mbi_hash, user_hicn_hash, err_msg)
+        log_dict["status"] = "FAIL"
+        log_dict["mesg"] = err_msg
+        logger.info(log_dict)
         raise BBMyMedicareCallbackCrosswalkCreateException(err_msg)
 
     if User.objects.filter(username=username).exists():
         err_msg = "user already exists"
-        log_create_beneficiary_record(auth_flow_dict, "FAIL", username, fhir_id, user_mbi_hash, user_hicn_hash, err_msg)
+        log_dict["status"] = "FAIL"
+        log_dict["mesg"] = err_msg
+        logger.info(log_dict)
         raise ValidationError(err_msg, username)
 
     if Crosswalk.objects.filter(_user_id_hash=user_hicn_hash).exists():
         err_msg = "user_hicn_hash already exists"
-        log_create_beneficiary_record(auth_flow_dict, "FAIL", username, fhir_id, user_mbi_hash, user_hicn_hash, err_msg)
+        log_dict["status"] = "FAIL"
+        log_dict["mesg"] = err_msg
+        logger.info(log_dict)
         raise ValidationError(err_msg, user_hicn_hash)
 
     # If mbi_hash is not NULL, perform check for duplicate
     if user_mbi_hash is not None:
         if Crosswalk.objects.filter(_user_mbi_hash=user_mbi_hash).exists():
             err_msg = "user_mbi_hash already exists"
-            log_create_beneficiary_record(auth_flow_dict, "FAIL", username, fhir_id, user_mbi_hash, user_hicn_hash, err_msg)
+            log_dict["status"] = "FAIL"
+            log_dict["mesg"] = err_msg
+            logger.info(log_dict)
             raise ValidationError(err_msg, user_hicn_hash)
 
     if fhir_id and Crosswalk.objects.filter(_fhir_id=fhir_id).exists():
         err_msg = "fhir_id already exists"
-        log_create_beneficiary_record(auth_flow_dict, "FAIL", username, fhir_id, user_mbi_hash, user_hicn_hash, err_msg)
+        log_dict["status"] = "FAIL"
+        log_dict["mesg"] = err_msg
+        logger.info(log_dict)
         raise ValidationError(err_msg, fhir_id)
 
     with transaction.atomic():
-        user = User(username=username,
-                    first_name=first_name,
-                    last_name=last_name,
-                    email=email)
+        user = User(
+            username=username, first_name=first_name, last_name=last_name, email=email
+        )
         user.set_unusable_password()
         user.save()
-        Crosswalk.objects.create(user=user,
-                                 user_hicn_hash=user_hicn_hash,
-                                 user_mbi_hash=user_mbi_hash,
-                                 fhir_id=fhir_id,
-                                 user_id_type=user_id_type)
+        Crosswalk.objects.create(
+            user=user,
```

Comment:
Adding crosswalk logging continued:

```suggestion
        cw = Crosswalk.objects.create(
            user=user,
```